### DISCUSSION TITLE
Reporting a stalled action

### DISCUSSION THREAD

**Discussion by alias-rahil** [community/community#26402] [Actions]
<p>Hello, this action is stalled. I have tried cancelling it but it won’t stop. Some internal staff/engineer can probably cancel it manually. Thanks.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/ashikka/lfs/actions/runs/340307726" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/59060219/181097504-d6a19da5-b792-4b81-a39a-97f37dfdb56b.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/ashikka/lfs/actions/runs/340307726" target="_blank" rel="noopener nofollow ugc">ashikka/lfs</a></h3>

<p>building LFS inside docker container. Contribute to ashikka/lfs development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/alias-rahil">@alias-rahil</a>,</p>
<p>I check the workflow run (<strong>340307726</strong>) in your repository via the GitHub Actions API “<a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#get-a-workflow-run" rel="noopener nofollow ugc"><strong>Get a workflow run</strong></a>” and “<a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-jobs-for-a-workflow-run" rel="noopener nofollow ugc"><strong>List jobs for a workflow run</strong></a>”.</p>
<p>It’s very odd that the status of the step “<strong>Build</strong>” still is “<strong>in_progress</strong>”, this cases the job “<strong>update</strong>” and the workflow run also are “<strong>in_progress</strong>”. Normally, the job should have been cancelled by timeout if it has executed 6 hours (see <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits" rel="noopener nofollow ugc">here</a>).</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Apologies for the wait here. I’ve been chasing down a solution.</p>
<p>We’re hoping this will be cleared today. It looks as though there was a change made to the repository while the job was running that may have caused it to glitch.</p>
